low-level/framework/base/sspl_constants.py

```diff
@@ -65,6 +65,9 @@
 CONSUL_PORT = '8500'
 CONSUL_ERR_STRING = '500 No cluster leader'
 
+# required only for support bundle.
+SB_REQUEST_TIMEOUT = 1200
```

Comment:
so 20 mins we expect max for a support bundle request to finish .. may be add comment where it usually takes time .. in support bundle script code

Reply:
Done